OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -284,4 +295,4 @@ cd "$(SolutionDir)"</PostBuildEvent>
   <Target Name="AfterBuild">
   </Target>
   -->
-</Project>
+</Project>
```

Comment:
I faintly remember this was a topic in the past, but I don't remember whether we wanted to have a newline at the end of .csproj or not. Just pointing it out so it doesn't get overlooked by other reviewers.